app/frontend/src/registry/components/people/ApplicationAdd.vue

```diff
@@ -1,218 +1,156 @@
-          removalDate: ''
-        }
-      },
-      issuerOrgs: [
-        'Canadian Groundwater Association',
-        'Province of BC',
-        'N/A'
-      ],
-      qualCodes: [
-        {
-          activity: 'DRILL',
-          quals: ['WAT', 'MON', 'RECH', 'DEWAT', 'REM', 'GEO']
-        },
-        {
-          activity: 'GEOTECH',
-          quals: ['MON', 'REM', 'GEO']
-        },
-        {
-          activity: 'GEOTHERM',
-          quals: ['CLOS']
-        },
-        {
-          activity: 'PUMP',
-          quals: []
-        }
-      ],
-      qualType: 'DRILL'
+      qualificationForm: {
+        subactivity: null,
+        primary_certificate_no: null,
+        primary_certificate: null,
+        status_set: [
+          {
+            effective_date: null,
+            status: 'P'
+          }
+        ],
+        qualifications: []
+      }
     }
   },
-  computed: {
-    quals () {
-      const quals = []
-      const activity = this.qualCodes.find(item => item.activity === this.qualType)
-      return activity ? activity.quals : quals
+  watch: {
+    // Watching the entire object is expensive, but we need some way of notifying the parent.
+    computedQualificationForm: {
+      handler: function (val, oldVal) {
+        this.$emit('input', val)
+      },
+      deep: true
     }
   },
-  watch: {
-    qualType: function () {
-      const qualList = this.quals.map(item => item.toLowerCase())
-      for (let key in this.editClassification.qualCodes) {
-        this.editClassification.qualCodes[key] = false
+  methods: {
+    changedClassification (value) {
+      const match = this.subactivityMap.filter(item => item.registries_subactivity_code === value)[0]
+      this.qualificationForm.qualifications = match.qualification_set.map(item => item.well_class)
+    },
+    ...mapActions([
+      'fetchDrillerOptions'
+    ])
+  },
+  computed: {
+    computedQualificationForm: function () {
+      // We need to transform the bound dates to something that is acceptable to the API without affecting
+      // the values to which the form are bound.
+      // Make a deep copy, and transform the date fields.
+      const transformed = JSON.parse(JSON.stringify(this.qualificationForm))
+      transformed.status_set.forEach((status) => { status.effective_date = status.effective_date && status.effective_date.length >= 10 ? status.effective_date.substring(0, 10) : status.effective_date })
+      return transformed
+    },
+    ...mapGetters([
+      'loading'
+    ]),
+    formOptions () {
+      let result = {
+        issuer: [{value: null, text: 'Please select an option'}],
+        classifications: [],
+        qualifications: []
       }
-      for (let i = 0; i < qualList.length; i++) {
-        if (qualList[i] in this.editClassification.qualCodes) {
-          this.editClassification.qualCodes[qualList[i]] = true
-        }
+      if (this.$options.propsData.activity in this.$store.getters.drillerOptions) {
```

Comment:
Since we are already using mapGetters (in the computed properties) you could use `...mapGetters(['loading', 'drillerOptions'])` and then reference this.drillerOptions